internal/configuration/configuration_test.go

```diff
@@ -16,16 +16,16 @@ func TestGetConfigWhitelist(t *testing.T) {
 		panic(err)
 	}
 
-	expected := map[string]*configInfo{
+	expected := map[string]*configFieldInfo{
 		"disable_feature_flags": {name: "DisableFeatureFlags", kind: reflect.Bool},
```

Comment:
What's the UpperCamelCase formatting for?

Reply:
Oh if you mean the `name:` field values, they have to match the `config` struct name exactly, so since the `config` fields are exported, these have to UpperCamelCase.

Reply:
I think it is better to include the entire configuration field's information (field name in Go + type), as if any of these fields are renamed or their type is changed, this test would fail (which is good in this case!). 

If a modification happens to any of these fields, and we just check that the JSON tags are in the whitelist, this test would still pass, but using the `describe/update` subcommand could panic when trying to retrieve the `Value` of the field by name.